docs/proposals/kubelet-oci-runtime.md

```diff
@@ -0,0 +1,87 @@
+Kubelet OCI runtime integration
+===========================================================
+Authors: Mrunal Patel (@mrunalp), Vishnu Kannan (@vishh)
+
+## Abstract
+This proposal aims to introduce support for OCI Runtime Specification compatible runtimes in Kubelet.
+
+## Motivation
+Kubelet should support runc and OCI as a container runtime to support Open Standards. Runc is the OCI compliant
```

Comment:
I would argue that in this case, we mean that we implement the `runtime-spec`. But yes, calling it "the OCI compliant implementation" is inaccurate and not helpful since there are already others.